src/content/docs/apm/agents/manage-apm-agents/azure/monitoring-azure-app-service.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Monitoring Azure app service
+tags:
+  - Azure
+  - Agents
+  - Manage APM agents
+metaDescription: Information on how to monitor Azure app services with New Relic agents.
+redirects:
+freshnessValidatedDate: never
+---
+
+## Overview of Azure app service [#overview]
+
+Azure App Service is a fully managed Platform as a Service (PaaS) offering from Microsoft Azure that enables developers to build, deploy, and scale web applications, mobile backends, and APIs quickly and efficiently across various programming languages and frameworks. Key features include automatic scaling to meet changing demand, continuous deployment through integration with Git repositories like GitHub and Azure DevOps, built-in security and compliance measures, and the ability to deploy applications globally for improved performance and availability.
```

Comment:
Are we including references to the Container service that's also available to certain APM agents? @elucus

Reply:
Yes, for Python only right now